### DISCUSSION TITLE
Forgot my password functionality

### DISCUSSION THREAD

**Discussion by AleDiBernardo** [adonisjs/core#4821] [Help]
Hi, Has been implemented a `forgot my password` functionality in the documentation?

**Comment by RomainLanz** [comment]
Hey @AleDiBernardo! 👋🏻

There's no official implementation for a "Forgot Password" feature. Are you encountering any specific challenges while working on it in your project?

If you're uncertain about the best practices for designing this feature, I'd recommend checking out the [OWASP guide](https://cheatsheetseries.owasp.org/cheatsheets/Forgot_Password_Cheat_Sheet.html) for helpful insights.

**Reply by AleDiBernardo** [reply]
Yes, I was unsure about the best practices to follow. 

Thanks for the reply, I will take a look at the guide right away 😄
